.azure-pipelines/gpu-tests.yml

```diff
@@ -20,8 +20,6 @@ jobs:
   - job: pytest
     strategy:
       matrix:
-        'PyTorch - LTS':
-          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8"
```

Comment:
Let's then test at least 1.9

Reply:
One advantage is to reduce the CI impact now. We can do this in a follow-up